File: calculator/calculator.rb
class Calculator
    def add(x, y)
      x + y
    end
    
    def subtract(x, y)
      x - y
    end
    
    def multiply(x, y)
      x * y
    end
    
    def divide(x, y)
      raise ArgumentError, "Cannot divide by zero" if y == 0
      x.to_f / y
    end
  end

File: calculator/calculator.js
class Calculator {
    add(x, y) {
        return x + y;
    }

    subtract(x, y) {
        return x - y;
    }
    
    multiply(x, y) {
        return x * y;
    }
    
    divide(x, y) {
        if (y === 0) {
            throw new Error("Cannot divide by zero");
        }
        return x / y;
    }
}
const calc = new Calculator();
console.log(calc.subtract(10, 4));  // Output: 6